### PR TITLE
Working

### PR DESCRIPTION
**Pull request by BSGreaves**
@drdreed

Dynamic Cards ready for feedback

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        <meta charset="utf-8">
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Dynamic Cards</title>
+        <link rel="stylesheet" href="cards.css">
+        <link rel="author" href="humans.txt">
+    </head>
+    <body>
+    	<header>
+	    	<h1>Dynamic Cards</h1>
```
Comment (DevDreed) [inline index.html]: Some weird indents here.  Not a big deal just readability.

**Review by DevDreed — CHANGES_REQUESTED** [review]
Fix up index.html formatting.

**Comment by BSGreaves** [comment]
@drdreed fixed indents, readme complete, thx.

**Review by DevDreed — APPROVED** [review]
👍

**Pull request merged**